charts/logzio-apm-collector/values.yaml

```diff
@@ -0,0 +1,662 @@
+    # If not set a name is generated using the fullname template
+    # Can be used in combination with presets that create a cluster role binding.
+    name: ""
+
+service:
+  # Enable the creation of a Traces Collector Service.
+  enabled: true
+
+  type: ClusterIP
+  # type: LoadBalancer
+  # loadBalancerIP: 1.2.3.4
+  # loadBalancerSourceRanges: []
+
+  # Annotations to add to the Service.
+  annotations: {}
+
+  ## By default, Service will be created setting 'internalTrafficPolicy: Cluster'
+  ## unless other value is explicitly set.
+  ## Setting 'internalTrafficPolicy: Cluster' on a daemonset is not recommended (in such case, use 'internalTrafficPolicy: Local')
+  # internalTrafficPolicy: Cluster
+
+  ## By default, Service of type 'LoadBalancer' will be created setting 'externalTrafficPolicy: Cluster'
+  ## unless other value is explicitly set.
+  ## Possible values are Cluster or Local (https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip)
+  # externalTrafficPolicy: Cluster
+
+spmService:
+  # Only generated if spm.enabled is set to true.
+  type: ClusterIP
+
+  # Annotations to add to the Service.
+  annotations: {}
+
+  ## By default, Service will be created setting 'internalTrafficPolicy: Cluster'
+  ## unless other value is explicitly set.
+  ## Setting 'internalTrafficPolicy: Cluster' on a daemonset is not recommended (in such case, use 'internalTrafficPolicy: Local')
+  # internalTrafficPolicy: Cluster
+
+  ## By default, Service of type 'LoadBalancer' will be created setting 'externalTrafficPolicy: Cluster'
+  ## unless other value is explicitly set.
+  ## Possible values are Cluster or Local (https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip)
+  # externalTrafficPolicy: Cluster
+
+# Configure HPA for Traces Collector.
+# Make sure that the `service.type` is `ClusterIP` to utilize K8S ability to automatically distribute traffic across all pod replicas
+autoscaling:
+  # Enable the creation of HPA for autoscaling.
+  enabled: false
+  # Annotations to add to the HPA.
+  annotations: {}
+  # Control autoscaling scale
+  minReplicas: 1
+  maxReplicas: 10
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
+
+# Configure VPA for SPM Collector.
+# Vertical scaling is used instead of horizontal scaling to ensure the accuracy of SPM aggregations.
+# Note: This feature requires the VPA Custom Resource Definitions (CRDs) to be installed. 
+# Installation guide: https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/docs/installation.md
+spmAutoscaling:
+  # Enable the vertical scaling
+  enabled: false
+  # Annotations to add to the HPA.
+  annotations: {}
+  # Control scaling limits
+  minAllowed:
+    cpu: 50m
+    memory: 70Mi
+  maxAllowed:
+    cpu: 150m
+    memory: 250Mi
+
+# Configuration for ports
+ports:
```

Comment:
ports for traces collector? or spm collector?

Reply:
I used it for both, but I can split them to be more precise and reduce the amount of ports for SPM since we need less open ports there 👍🏼